src/plugins/redirections/crossing/checkCrossingRedirections.ml

```diff
@@ -0,0 +1,232 @@
+open Lintshell
```

Comment:
Is it the right place for these general purpose functions over CSTs? Shouldn't they moved to libmorbig?

Reply:
If you are talking about these `io_redirect_list_of_cmd_prefix` functions, I totally agree. That would also allow for factorization of code between shstats and lintshell in many cases.

Reply:
Yes.